tests/e2e/core-tests/specs/shopper/front-end-variable-product-updates.test.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable jest/no-export, jest/no-disabled-tests */
+/**
+ * Internal dependencies
+ */
+const {
+	shopper,
+	merchant,
+	createVariableProduct,
+} = require( '@woocommerce/e2e-utils' );
+
+let variablePostIdValue;
+
+const cartDialogMessage = 'Please select some product options before adding this product to your cart.';
+
+const runVariableProductUpdateTest = () => {
+	describe('Update variable product',() => {
+		beforeAll(async () => {
+			await merchant.login();
+			variablePostIdValue = await createVariableProduct();
+			await merchant.logout();
+		});
+
+		it('can change variable attributes to the same value', async () => {
```

Comment:
Again maybe need to make clearer we are referring to a Shopper interacting with the variation.

Reply:
Agree! I updated the `it()` statements to include `shopper` so it's clearer :)